src/build/partition.jl

```diff
@@ -469,14 +545,22 @@ end
 
 """
 `y(yr,g)`, gross output
+"Move household supply of recycled goods into the domestic output market
+from which some may be exported. Net out margin supply from output."
 
 ```math
-\\tilde{y}_{yr,g} = \\sum_{s}\\tilde{ys}_{yr,s,g} - \\sum_{m}\\tilde{ms}_{yr,g,m}
+\\tilde{y}_{yr,g} = \\sum_{s}\\tilde{ys}_{yr,s,g} + \\tilde{fd}_{yr,g} - \\sum_{m}\\tilde{ms}_{yr,g,m}
 ```
 """
 function _partition_y0!(d::Dict, set::Dict)
     println("  Partitioning y0, gross output")
-    d[:y0] = combine_over(d[:ys0], :s) + d[:fs0] - combine_over(d[:ms0], :m)
+
+    d[:y0] = if d[:sector]==:summary
+        combine_over(d[:ys0], :s) + d[:fs0] - combine_over(d[:ms0], :m)
+    elseif d[:sector]==:detail
+        combine_over(d[:ys0], :s) + d[:fs0]
+    end
```

Comment:
**[future] for EEM**. should these be different for summary/detail-level?

Reply:
If I recall, margins associated with the sets in question for disaggregating are zero --- so it might not actually matter (may be wrong on this). If you were building a benchmark from scratch using the detail data, the summary level method is correct. But if using detail data to update pre-existing benchmark, it may not matter. Definitely something to flag for quality check.